### ISSUE TITLE
REST API endpoints specified in .proto files do not work

### ISSUE THREAD

**Issue by gabbifish** [dedis/drand#79]
`drand/protobuf/drand/client.proto` defines services with a RESTful proxy on top of some drand services. For example,
```
service Info {
  rpc DistKey(DistKeyRequest) returns (DistKeyResponse) {
    option (google.api.http) = {
      get: "/info/dist_key"
    };
  }
}
```
This means that one should be able to request resources from a drand beacon with the following url:
```
http://<address>/info/dist_key
```
However, when I attempt to access the distributed key via http on a local cluster of drand nodes (using `localhost:<port>/info/dist_key`, I get a 404 Not Found. It seems that this RESTful proxy is not working.

**Comment by gabbifish** [comment]
Fix PR in #81

**Comment by gabbifish** [comment]
PR #81 merged!
